internal/distro/rhel8/distro.go

```diff
@@ -673,9 +673,6 @@ func New() distro.Distro {
 			"iwl2000-firmware", "iwl2030-firmware", "iwl3160-firmware", "iwl5000-firmware",
 			"iwl5150-firmware", "iwl6000-firmware", "iwl6050-firmware", "iwl7260-firmware",
 		},
-		excludedPackages: []string{
-			"subscription-manager",
-		},
```

Comment:
NB: This is changing the "stable" 8.3 distro. I guess that is want you wanted, just wanted to make sure that it is clear ;)

Reply:
Thanks for pointing this out @gicmo.

8.3 is final. Please don't change it anymore.

Reply:
This is indeed confusing, we should rename `rhel8` to `rhel83` to avoid future confusion. If you want to change 8.4, then patch `rhel84/distro.go`.

Reply:
perhaps 8.3 can be left unchanged - let me check

Reply:
Yep, 8.3 can stay as it is, rhc will only be in 8.4+

Reply:
fixed, thx all